front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import cx from 'classnames';
+
+import type {
+  PathPropertiesFormatted,
+  SimulationResponseSuccess,
+} from 'applications/operationalStudies/types';
+import type { TrainScheduleResult } from 'common/api/generatedEditoastApi';
+import type { OperationalPointWithTimeAndSpeed } from 'modules/trainschedule/components/DriverTrainScheduleV2/types';
+import type { PathStep } from 'reducers/osrdconf/types';
+import { NO_BREAK_SPACE } from 'utils/strings';
+
+import useOutputTableData from './hooks/useOutputTableData';
+import TimesStops from './TimesStops';
+import { TableType, type PathWaypointRow } from './types';
+
+type TimesStopsOutputProps = {
+  simulatedTrain: SimulationResponseSuccess;
+  pathProperties: PathPropertiesFormatted;
+  operationalPoints: OperationalPointWithTimeAndSpeed[];
+  selectedTrainSchedule: TrainScheduleResult;
+  startTime: string;
+  pathSteps: PathStep[];
+  pathLength?: number;
+};
+
+const TimesStopsOutput = ({
+  simulatedTrain,
+  pathProperties,
+  operationalPoints,
+  selectedTrainSchedule,
+  startTime,
+  pathSteps,
+  pathLength,
+}: TimesStopsOutputProps) => {
+  const enrichedOperationalPoints = useOutputTableData(
+    simulatedTrain,
+    pathProperties,
+    operationalPoints,
+    selectedTrainSchedule,
+    startTime,
+    pathLength
+  );
+  return (
+    <TimesStops
+      allWaypoints={enrichedOperationalPoints}
+      pathSteps={pathSteps}
+      startTime={startTime}
```

Comment:
Actually, you can use directly the start time of the selectedTrain. You just need to transform it to the locale timeZone.
- add a new utils function in utils/date.ts
```
export const isoUtcStringToLocaleDateString = (isoUtcString: string) => {
  return dayjs(isoUtcString).local().format();
};
```
- give it to TimesStops
```suggestion
      startTime={isoUtcStringToLocaleDateString(selectedTrainSchedule.start_time)}
```

Reply:
done